src/bridge/cockpitinternalmetrics.c

```diff
@@ -73,6 +73,7 @@ static MetricDescription metric_descriptions[] = {
   { "cpu.core.user",   "millisec", "counter", TRUE, CPU_SAMPLER },
   { "cpu.core.system", "millisec", "counter", TRUE, CPU_SAMPLER },
   { "cpu.core.iowait", "millisec", "counter", TRUE, CPU_SAMPLER },
+  { "cpu.temperature", "celsius", "counter", FALSE, CPU_SAMPLER },
```

Comment:
This is not a `counter` value.

I also feel like this is an awkward fit.

All of the existing CPU samples are counters in milliseconds, and are available per-core and "basic" summary values.

I'd prefer to see a new sampler created for sensor information.  We might imagine reporting other temperatures in the future...